rust/benchmarks/src/bin/tpch.rs

```diff
@@ -363,9 +363,9 @@ fn create_logical_plan(ctx: &mut ExecutionContext, query: usize) -> Result<Logic
             from
                 lineitem
             where
-                l_shipdate >= '1994-01-01'
-                and l_shipdate < '1995-01-01'
-                and l_discount between 0.06 - 0.01 and 0.06 + 0.01
+                l_shipdate >= date '1994-01-01'
+                and l_shipdate < date '1995-01-01'
+                and l_discount > 0.06 - 0.01 and l_discount < 0.06 + 0.01
```

Comment:
Shouldn't this have between that was also added recently? @seddonm1

Reply:
yes, sorry i will fix this.

Reply:
Actually, no. The raw query as per TPC-H does not use between for that clause:

```sql
where l_shipdate >= date '[DATE]' 
and l_shipdate < date '[DATE]' + interval '1' year
and l_discount between [DISCOUNT] - 0.01 and [DISCOUNT] + 0.01
```

Reply:
I see a between on the last line?

Reply:
Oh yes, sorry I thought you meant add it to the `l_shipdate` component. Yes, i will fix that.